meet-data/gpa/1702/entries.csv

```diff
@@ -0,0 +1,326 @@
+Place,Tested,Equipment,Event,WeightClassKg,Sex,Name,Division,BirthYear,Age,BodyweightKg,Country,Squat1Kg,Squat2Kg,Squat3Kg,Squat4Kg,BestSquatKg,Bench1Kg,Bench2Kg,Bench3Kg,Bench4Kg,BestBenchKg,Deadlift1Kg,Deadlift2Kg,Deadlift3Kg,Deadlift4Kg,BestDeadliftKg,TotalKg,Team
```

Comment:
The "Team" column isn't their team -- that's actually the name of their coach. We don't track that; you can delete it.

For the Country column, we want just "Russia" instead of "Russia/Tula/Tulskaya oblast".